030-ConversationalAI/Student/Challenge1-QnA.md

```diff
@@ -0,0 +1,37 @@
+# Challenge 1 - QnA Maker
+[< Previous Challenge](./Challenge0-Setup.md) - **[Home](../README.md)** - [Next Challenge>](./Challenge2-LUIS.md)
+## Introduction
+QnA maker is a no code experience that allows you to create a natural conversational layer over you data. It is used to find the most appropriate answer for any input from your custom knowledge base of information. We're going to be building out a QnA based off of the FSI Industry Use Case Document that is in the Student folder of this Github. However you can build QnA out of a website, excel document, FAQ, etc. Our QnA Knowledge base will deal with the ESG portion of our project. You can imagine that a Investment Manager is looking at their stocks and considering what to buy. They can interact with the bot to ask for the ESG rating of a specific company and then dive deeper into  the Company belief's for specific Environmental topics. In this hack we're just focusing on Microsoft, but building out a enterpris bot you would have a much larger knowledge base to pull from.
+
+
+## Description
+
+1. You are going to upload two different sources into a QnA knowledge base. You should be doing this through Bot Composer and **not** through the QnaMaker UI. 
+
+	- Create a new QNA KB using [Microsoft Investor FAQ](https://www.microsoft.com/en-us/Investor/FAQ.aspx)
+	- Also upload thhe  [FSI Use Case document](./Resources/T20-FSI-ESG-BOT-ACS-IndustryUseCase.docx). 
```

Comment:
See my other comment next to Gino's.   When linking to a document, use the raw link so the browser downloads the file instantly instead of directing the user to the repo on GitHub.  

This serves two purposes:
1) reduces the # of clicks for the user
2) more importantly, helps our scheme to obfuscate the What The Hack repo from students during the actual hack!  If they find the repo, it's easier for them to cheat.